pkg/logger/logger_test.go

```diff
@@ -33,16 +31,6 @@ func setupLogEnv() {
 	lumberjackLogger = tryNewLumberJackLogger()
 }
 
-func TestGetLumberHook(t *testing.T) {
```

Comment:
Please replace this unit test with one for the `WrapCoreOpt`.

Reply:
This test had zero value and a test for `WrapCoreOpt` would likely also have zero value.

Reply:
That's fair. I agree that the original test didn't really do anything and I suppose a new one would just be validating the config.